src/index.js

```diff
@@ -75,9 +79,7 @@ const happypack = (
   return group(blocks.map(block =>
     Object.assign(happifyBlock(block, options), {
       pre: block.pre,
-      post: []
-        .concat(block.post || [])
-        .map(postHook => happifyBlock(postHook, options)),
+      post: block.post,
```

Comment:
Wasn't clear what the old post hook processing was doing, or how to adapt it for new API, so I took it out.  @diegohaz happy to fix if you can explain the requirements.

Reply:
It's happifying the blocks' post hooks. I suppose that's got to stay like that:
```js
post: (block.post || []).map(postHook => happifyBlock(postHook, options))
```

But maybe @diegohaz knows more.